pkg/backend/display/testdata/not-truncated/webserver-userdata.json.stdout.txt

```diff
@@ -41,9 +41,6 @@
 <{%fg 1%}>          - echo "Hello, World!" > index.html<{%reset%}>
 <{%fg 2%}>          + echo "Hello, Pulumi!" > index.html<{%reset%}>
 <{%reset%}>            nohup python -m SimpleHTTPServer 80 &<{%reset%}>
-<{%reset%}><{%!r(MISSING)eset%!}(MISSING)>    --outputs:--<{%!r(MISSING)eset%!}(MISSING)>
-<{%fg 3%}>  ~ publicHostName: <{%reset%}><{%fg 1%}>"ec2-34-211-56-110.us-west-2.compute.amazonaws.com"<{%reset%}><{%fg 3%}> => <{%reset%}><{%fg 2%}>output<string><{%reset%}><{%fg 3%}>
-<{%reset%}><{%fg 3%}>  ~ publicIp      : <{%reset%}><{%fg 1%}>"34.211.56.110"<{%reset%}><{%fg 3%}> => <{%reset%}><{%fg 2%}>output<string><{%reset%}><{%fg 3%}>
```

Comment:
This test also encountered the issue, and I suppose this diff demonstrates the fix – but this test's purpose is unrelated to the issue I address in this PR, so I think it's better to add a new test to specifically verify my patch.